setup.py

```diff
@@ -57,6 +57,7 @@ def run_tests(self):
         'six>=1.10.0',
         'graphql-core>=0.4.9',
         'graphql-relay==0.3.3',
+        'iso8601==0.1.11',
```

Comment:
I will either have the `iso8601` to be in `graphql/core/pyutils/iso8601.py` and import from there, or to require a generic version of `iso8601` in setup.py (generic version == no version specified).

The reason for that is any other package could be already using an older or newer version of `iso8601` and when installing `graphene` package it will trigger a incompatibility error as requires a specific version of it. (That's the reason Django includes a lot of third-party libraries in their own codebase).